views/layouts/main.handlebars

```diff
@@ -13,10 +13,17 @@
   <script defer src="/l20n/dist/web/l20n.js"></script>
 </head>
 <body>
-  <div class="send-logo">
-    <img src="/resources/send_logo.svg"/>
-    <img src="/resources/send_logo_type.svg"/>
-  </div>
+  <header class="header"> 
+    <div class="send-logo">
+      <img src="/resources/send_logo.svg"/>
+      <h1 class="site-title">Send</h1>
+      <div class="site-subtitle">
+        <a href="https://testpilot.firefox.com">Firefox Test Pilot</a>
```

Comment:
Should this have a `target="_blank"` and/or would "Firefox Test Pilot" ever be localized? I'm assuming not, per [**mozilla/testpilot** /locales/zh-CN/app.ftl:7-10](https://github.com/mozilla/testpilot/blob/3c6d2670e7165b3ac38e61cb5243a6284e3cb149/locales/zh-CN/app.ftl#L7-L10), but figured I'd verify (considering it's in the TestPilot .ftl files).